bst_plugin/process_nst_compute_ttest.m

```diff
@@ -228,15 +228,19 @@
 
 %         % Saving the cB Matrix
     sOutput_b = db_template('matrixmat');
-    sOutput_b.Value           = B.Value';
+    sOutput_b.F           = B.Value';
     sOutput_b.Comment     = [contrast ' B' ];
-    sOutput_b.Description = B.Description;  
-    sOutput_b.ChannelFlag =  B.ChannelFlag;   
-    sOutput_b.Type         = 'data';
+    sOutput_b.Description = contrast;  
+    sOutput_b.ChannelFlag =  B.ChannelFlag;
+    sOutput_b.Time         = [1];
```

Comment:
Maybe we'd have to change the time axis to a add a dummy second time point.
I got some visualization errors sometimes when using only one time point.
Anyway, internally brainstorm will create a second time point I think.